policy/policy.example.agent.json

```diff
@@ -0,0 +1,35 @@
+{
+    "description": "hello there, security policies are fun!",
+    "rules": [
+      {
+        "is_stateful": true,
+        "ports": [
+          80,
+          443
+        ],
+        "protocol": "TCP"
+      }
+    ],
+    "peers": [
+      {
+        "segment_network_id": 2
+      }
+    ],
+    "direction": "ingress",
+    "applied_to": [
+      {
+        "segment_network_id": 1,
+        "tenant_network_id": 1
+      }
+    ],
+    "datacenter" : {
```

Comment:
I had a question on this in the paninetworks account.

I was thinking about that. Should we pass the "bit pattern spec" in with each policy? That would make it nicely self contained. However, we have traditionally considered the bit-field definitions as a configuration item, which therefore doesn't have to be passed around with every operation.

Different components in the system need to agree on this sort of configuration. IPAM, Tenant service, etc. It seems to me therefore that it's better to have all of them read it from the same config location, rather then passing it around with each request (which could result in accidentally spreading a mis-matched view of the world to different components).

What do you think?